app/src/main/java/io/intrepid/contest/screens/contestcreation/categorieslist/CategoriesListFragment.java

```diff
@@ -98,4 +98,16 @@ public Category getDefaultCategory(int categoryNameRes, int categoryDescriptionR
         String categoryDescription = getString(categoryDescriptionRes);
         return new Category(categoryName, categoryDescription);
     }
+
+    @Override
+    public void setNextEnabled(boolean enabled) {
+        ((EditContestContract) getActivity()).setNextEnabled(enabled);
+    }
+
+    @Override
+    public void onFocus() {
+        if (presenter != null) {
```

Comment:
It occurs when we move back and forth from one fragment to the other. Since we're just calling fragment.onFocus() from another presenter, we don't necessarily know if the fragment's presenter has been instantiated yet. I feel like there's a better workaround to this, its just not coming to me